backend/battle/forms.py

```diff
@@ -83,15 +83,14 @@ def clean(self):
                 cleaned_data['pokemon_3']
             ]
         )
-
-        cleaned_data['pokemon_1_object'] = get_pokemon_object(cleaned_data['pokemon_1'])
-        cleaned_data['pokemon_2_object'] = get_pokemon_object(cleaned_data['pokemon_2'])
-        cleaned_data['pokemon_3_object'] = get_pokemon_object(cleaned_data['pokemon_3'])
+        cleaned_data['pokemon_1_object'] = Pokemon.objects.get(name=cleaned_data['pokemon_1'])
+        cleaned_data['pokemon_2_object'] = Pokemon.objects.get(name=cleaned_data['pokemon_2'])
+        cleaned_data['pokemon_3_object'] = Pokemon.objects.get(name=cleaned_data['pokemon_3'])
         return cleaned_data
 
     def save(self, commit=True):
         data = self.clean()
-        instance = super().save()
+        instance = Team.objects.create(battle=data['battle'], trainer=data['trainer'])
```

Comment:
Why can't you call super anymore?

Reply:
You're right, I changed this now, thanks!